crud.py

```diff
@@ -9,19 +9,6 @@
 # from loguru import logger
 
 
-async def check_lnaddress_update(username: str, id: str) -> bool:
```

Comment:
`check_lnaddress_update(username: str, id: str)` doesn't make sense since `id`/`link_id` is not changed during the update, and the couple `username` and `id` is expected to be *always* unique.